flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -151,6 +151,9 @@ jobmanager.web.port: 8081
 
 # Path to the Hadoop configuration directory.
 #
+# Warning: these keys are deprecated and will be removed in 1.5. Instead, use
```

Comment:
I see your question on a specific future version for removing these keys was not answered but I expect that it won't be for 1.5 if ever. I think we can leave this at the deprecation notice and recommendation to use `HADOOP_CONF_DIR` without stating a time for removal.